discord/voice_client.py

```diff
@@ -623,13 +623,33 @@ def get_ssrc(self, user_id):
             user_id
         ]
 
+    @overload
     def play(
         self,
         source: AudioSource,
         *,
         after: Callable[[Exception | None], Any] = None,
```

Comment:
Defaults are not needed in overloads, that aside, shouldn't this have been typed as optional all along?
```suggestion
        after: Callable[[Exception | None], Any] | None,
```

Reply:
Defaults are actually needed, without them mypy won't identify how to properly call the method. I agree about marking it optional though.

Reply:
Updated my suggestions.